test/e2e/validation_test.go

```diff
@@ -478,15 +478,21 @@ func testServicesConfigMap(t *testing.T) {
 	servicesConfigMapName := servicescm.NamePrefix + operatorVersion
 
 	// Ensure the windows-services ConfigMap exists in the cluster
-	_, err = tc.client.K8s.CoreV1().ConfigMaps(tc.namespace).Get(context.TODO(),
-		servicesConfigMapName, meta.GetOptions{})
-	require.NoErrorf(t, err, "error retrieving ConfigMap: %s", servicesConfigMapName)
+	t.Run("Services ConfigMap existance", func(t *testing.T) {
+		_, err = tc.client.K8s.CoreV1().ConfigMaps(tc.namespace).Get(context.TODO(), servicesConfigMapName,
+			meta.GetOptions{})
+		assert.NoErrorf(t, err, "error ensuring ConfigMap %s exists", servicesConfigMapName)
+	})
 
-	err = tc.testServicesCMRegeneration(servicesConfigMapName)
-	require.NoErrorf(t, err, "error ensuring ConfigMap %s is re-created when deleted", servicesConfigMapName)
+	t.Run("Services ConfigMap re-creation", func(t *testing.T) {
+		err = tc.testServicesCMRegeneration(servicesConfigMapName)
+		assert.NoErrorf(t, err, "error ensuring ConfigMap %s is re-created when deleted", servicesConfigMapName)
```

Comment:
Good catch, the test above it should also be a `require` as without the ConfigMap existing we can't test recreation, deletion, etc.

Reply:
Thought about this a bit more and since these are separate functions, a require will not stop the parent function `testServicesConfigMap`, only the subtests (each `t.Run` block). Though I may be mistaken.

> In case the ConfigMap re-creation fails, can WMCO still configure the instance without it?
Yep. Since WMCO does not need the services CM yet, I don't think we need to stop the e2e suite on failure here.

Reply:
I think we can make it mandatory in the test suite once we move the first service definition into the services ConfigMap. I'll leave the question of _should we continue e2e tests if the services ConfigMap does not exist in cluster_ to be revisited when it's applicable. I left a comment on the relevant story [WINC-738: Kubelet service is described in services ConfigMap](https://issues.redhat.com/browse/WINC-738) so we don't forget.